ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/version/withoutledgerid/VersionClient.scala

```diff
@@ -22,4 +23,24 @@ private[daml] final class VersionClient(service: VersionServiceStub) {
         new GetLedgerApiVersionRequest(ledgerIdToUse.unwrap)
       )
       .map(_.version)
+
+  def getApiFeatures(
+      ledgerIdToUse: LedgerId,
+      token: Option[String] = None,
+  )(implicit executionContext: ExecutionContext): Future[Seq[Feature]] =
+    LedgerClient
+      .stub(service, token)
+      .getLedgerApiVersion(
+        new GetLedgerApiVersionRequest(ledgerIdToUse.unwrap)
+      )
+      .map(_.features.toList.flatMap(VersionClient.fromProto))
+}
+
+private[daml] object VersionClient {
+  // see also com.daml.platform.apiserver.services.ApiVersionService.featuresDescriptor
+  def fromProto(featuresDescriptor: FeaturesDescriptor): Seq[Feature] =
+    featuresDescriptor match {
+      case FeaturesDescriptor(userManagement, _ /* TODO: expose experimentalFeatures */ ) =>
```

Comment:
```suggestion
      // Note that we do not expose experimental features here, as they are used for internal testing only
      // and do not have backwards compatibility guarantees. (They should probably be named 'internalFeatures' ;-)
      case FeaturesDescriptor(userManagement, _ ) =>
```

Reply:
Thanks for the suggestion! Added to the PR.